<launch>
    <node name="local_task_manager" pkg="ltm" type="ltm" output="screen" />
    <node name="graphical_user_interface" pkg="gui" type="gui.py" output="screen" />
    <node name="dialogue_manager" pkg="dialogue" type="dialogue.py" output="screen" />
</launch>